src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  white-space: nowrap;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  position: relative;
+}
+
+.nav__list {
+  display: flex;
+  z-index: 1;
```

Comment:
z-index is not working with **position: static**
( `static`  is a default value for the position )

Reply:
Fixed